platform/File.roc

```diff
@@ -39,7 +38,7 @@ import Host
 ## **OutOfMemory** - An operation could not be completed, because it failed to allocate enough memory.
 ##
 ## **Other** - A custom error that does not fall under any other I/O error kind.
-IOErr : InternalFile.IOErr
+Err : InternalIOErr.IOErr
```

Comment:
We've been using `Err` in all the modules, so this is to keep it consistent with all of those. I don't think it will be confused with `Result.Err`.

Reply:
I'll merge as is, if you don't agree I'm happy to revisit these in a follow up -- before we make our PI release.

Reply:
Yeah, weighing all the pros and cons, it seems way better to avoid the potential confusion if we can. For searches and refactors it seems very annoying to have to investigate if a use of `Err` is using `Result.Err` or this `Err`.